dataflow/build.gradle

```diff
@@ -59,3 +59,25 @@ task liveVariableTest(dependsOn: compileTestJava, group: 'Verification') {
         }
     }
 }
+
+task issue3447Test(dependsOn: compileTestJava, group: 'Verification') {
+    description 'Test issue 3447 test case for backward analysis.'
+    inputs.file('tests/issue3447/Test.java')
+
+    outputs.file('tests/live-variable/Out.txt')
+    outputs.file('tests/live-variable/Test.class')
+
+    delete('tests/issue3447/Out.txt')
```

Comment:
Does the test even produce an `Out.txt` file?

Reply:
It does. This test relies on [dataflow/src/test/java/livevar/LiveVariable.java](https://github.com/typetools/checker-framework/blob/master/dataflow/src/test/java/livevar/LiveVariable.java).

Reply:
You're just not comparing against a golden file, but the test suite still generates the Out.txt file. Sounds good.